src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -228,14 +227,26 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
           .format(format)
           .save(refreshTestLocation)
         hyperspace.refreshIndex(indexConfig.indexName)
+        val newIndexLocation = s"$systemPath/index_$format"
         indexCount = spark.read
-          .parquet(s"$systemPath/index_$format" +
+          .parquet(newIndexLocation +
             s"/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=1")
           .count()
 
         // Check if index got updated
         assert(indexCount == 3)
 
+        // Check if lastest log file is updated with newly created index files
+        val indexPath = PathUtils.makeAbsolute(newIndexLocation)
+        val logManager = IndexLogManagerFactoryImpl.create(indexPath)
+        val latestLog = logManager.getLatestLog()
+        assert(latestLog.isDefined && latestLog.get.isInstanceOf[IndexLogEntry])
+        val indexLog = latestLog.get.asInstanceOf[IndexLogEntry]
+        assert(indexLog.content.directories.nonEmpty)
+        assert(indexLog.content.directories.head.files.nonEmpty)
+        // TODO: change the below condition from `exists` to `forall` when _SUCCESS is removed.
```

Comment:
I guess we can just do `forall(f => f.name.startsWith("_") || f.name.contains("part-000")))`, so that when `_SUCCESS` is removed, this test would fail.

Reply:
thanks @imback82 , added with slight modification. `startsWith(_)` ->`contains(_SUCCESS)` because currently f.name is full path